code/modules/research/designs/equipment_designs.dm

```diff
@@ -280,3 +280,14 @@
 	build_type = PROTOLATHE
 	materials = list(MAT_METAL = 5000, MAT_PLASMA = 2500, MAT_TITANIUM = 500, MAT_BLUESPACE = 500)
 	category = list("Equipment")
+
+/datum/design/jawsoflife_rescue
+	name = "Rescue Jaws of Life"
+	desc = "The Rescue variant of the Jaws of Life, designed to quickly open doors in emergencies. Bigger and less versatile than the original."
```

Comment:
Shouldn't this mirror the item description (or vice versa)? They're slightly different, with this having `designed to quickly open doors in emergencies` and the main item desc having `Designed for prying open pesky doors in emergencies.`

Reply:
I think the purpose here is to specify that this variant can open them quicker than the normal set

Reply:
Ya, I just mean that the protolathe description is not the same as the item description :eyes:

Reply:
other items are not like that including the original jaws